tvision/include/tv/tvconfig.h

```diff
@@ -33,7 +33,7 @@ const int maxReplaceStrLen = 80;
 #include <tv/intl.h>
 
 #ifndef pid_t
-# define pid_t int
+# define pid_t unsigned int
 #endif
```

Comment:
Windows defines a PID to be a DWORD, which is an unsigned int, On Linux a PID only uses 22 bits, so an unsigned int should be adequate for both.

Reply:
I did a bit of research.  Turns out, for compatibility between 32-bit and 64-bit processes, only the lower 32-bits are used in hProcess.  But I have an alternative I'm working on, originating in your observation that hProcess is not used anywhere else, which I confirmed by searching the code.  In the call to CreateProcess, one of the return values is the PID in pi.dwProcessId, which is a DWORD, a 32-bit value.  So, rather than store hProcess in pidChild, we can store the actual PID instead.

Reply:
Please see latest commit, I believe I've addressed this issue.